ci-operator/step-registry/openshift-qe/concurrent-builds/openshift-qe-concurrent-builds-ref.yaml

```diff
@@ -0,0 +1,34 @@
+ref:
+  as: openshift-qe-concurrent-builds
+  from_image:
+    namespace: ci
+    name: ocp-qe-perfscale-ci
+    tag: latest
+  commands: openshift-qe-concurrent-builds-commands.sh
+  credentials:
+  - namespace: test-credentials # this entry injects the custom credential
+    name: ocp-qe-perfscale-es
+    mount_path: /secret
+  resources:
+    requests:
+      cpu: 100m
+      memory: 100Mi
+  env:
+  - name: BUILD_LIST
+    default: "1 8 15 30 45 50 75"
+    documentation: |-
+      Define the build list. 
+  - name: APP_LIST
+    default: "cakephp eap django nodejs"
+    documentation: |-
+      Define the app list.  
+  - name: POD_READY_THRESHOLD
+    default: "180000ms"
+    documentation: |-
+      Define the maximum number of pods to deploy on each labeled node.
+  - name: OUTPUT_FILE
+    default: "kube-burner-ocp.out"
```

Comment:
lets set the default here to be kube-burner.out since its not using the ocp verison yet. This way it'll properly read the output file to find the workload type

Reply:
#Meh 
I thought I changed that!